lib/rubocop/cop/sorbet/redundant_extend_t_sig.rb

```diff
@@ -40,7 +41,7 @@ def on_send(node)
           return unless extend_t_sig?(node)
 
           add_offense(node) do |corrector|
-            corrector.remove(node)
+            corrector.remove(range_by_whole_lines(node.source_range, include_final_newline: true))
```

Comment:
What if we did `range_with_surrounding_space(range: node.source_range)` instead?

Reply:
That's a lot cleaner. I'm guessing this will also take out additional newlines before and after?

Reply:
Ahh, that could be why 
<img width="714" alt="image" src="https://github.com/Shopify/rubocop-sorbet/assets/60748675/32f1f03a-9834-48f0-b09f-0dff7e77f6e2">

Reply:
[Looking at the docs](https://www.rubydoc.info/gems/rubocop/0.50.0/RuboCop/Cop/Util.range_with_surrounding_space), I wonder how many cases would be solved with:
```
range_with_surrounding_space(range: node.source_range, side: :right) 
```

Reply:
Ahh, not quite! This one is wrong:
```ruby
class SomeClass 
  extend T::Sig
end 

class SomeClass
  end
```

The indentation is kept because it doesn't go left. Though that becomes tough to manage since if it was anything other than `end` we would be fine with it.

Maybe it is best to keep it how it was and rely on the other cops to ensure the lines are cleaned up.

Reply:
Interesting, what if we did: `range_with_surrounding_space(range: node.source_range, side: :right)` and joined it with `range_with_surrounding_space(range: node.source_range, side: :left, newlines: false)`. So:
```ruby
add_range(
  range_with_surrounding_space(range: node.source_range, side: :left, newlines: false),
  range_with_surrounding_space(range: node.source_range, side: :right)
)
```

Reply:
Thanks for the inquiry, it's good to know what is going on at a deeper level.